mc_lib/observable.pyx

```diff
@@ -85,3 +86,31 @@ def block_stats(RealObservable obs):
     for j in range(arr.shape[0]):
         arr[j] = (v_av[j], v_err[j], v_size[j])
     return arr
+
+def write_observable_hdf5(obs, group):
+    """
+    Save RealObservable in hdf5 group
```

Comment:
Add an empty line to separate the short description (a single line) from the longer description, per numpydoc standard.